### PR TITLE
Repeatable text field in question group

### PR DESCRIPTION
**Pull request by elahti**

### CODE REVIEW

File: src/cljs/ataru/hakija/application_handlers.cljs [hunk 1]
```diff
@@ -263,14 +263,17 @@
 
 (defn set-question-group-row-amounts [db]
   (let [flattened-form-fields (cljc-util/flatten-form-fields (-> db :form :content))]
-    (reduce-kv (fn [db answer-key {:keys [value]}]
+    (reduce-kv (fn [db answer-key {:keys [value values]}]
```
Comment (heeboA) [inline src/cljs/ataru/hakija/application_handlers.cljs]: Extract large anon function?

File: src/cljs/ataru/hakija/hakija_readonly.cljs [hunk 2]
```diff
@@ -26,21 +26,28 @@
        (contains? field-descriptor :koodisto-source)))
 
 (defn text [field-descriptor application lang]
-  (let [answer ((answer-key field-descriptor) (:answers application))]
+  (let [answer            ((answer-key field-descriptor) (:answers application))
+        multi-values->:li (partial map-indexed (fn [idx value]
```
Comment (heeboA) [inline src/cljs/ataru/hakija/hakija_readonly.cljs]: Extract from let-block?

File: src/cljs/ataru/hakija/application_handlers.cljs (hunk 1 above)
Comment (elahti) [reply src/cljs/ataru/hakija/application_handlers.cljs]: Discussed this face-to-face. Decided to leave this as-is for now.

File: src/cljs/ataru/hakija/hakija_readonly.cljs (hunk 2 above)
Comment (elahti) [reply src/cljs/ataru/hakija/hakija_readonly.cljs]: Discussed this face-to-face. Decided to leave this as-is for now.

**Pull request merged**